runit-java8-bionic/collectd-jmx.conf

```diff
@@ -0,0 +1,133 @@
+LoadPlugin java
+
+<Plugin "java">
+  JVMArg "-Djava.class.path=/usr/share/collectd/java/collectd-api.jar:/usr/share/collectd/java/generic-jmx.jar"
```

Comment:
Is this going to get hit by the same bug as EN-31931 where collectd on on bionic is built with java 11 but we install java 8?

Reply:
Yes, it does. I've implemented a work-around that same way I plan on handling EN-31931 .